src/main/java/racingcar/view/InputParser.java

```diff
@@ -0,0 +1,5 @@
+package racingcar.view;
+
+public interface InputParser {
+    Object parse(String input);
+}
```

Comment:
Generic으로 짰으면 unsafe casting 문제 안생겼을텐데.
```suggestion
public interface InputParser<T> {
    T parse(String input);
}
```